api/app/src/routes/settings.ts

```diff
@@ -78,4 +103,48 @@ router.post(
   })
 );
 
+/** ---------------------------------------------------------------------------
+ * GET /settings/webhook
+ *
+ * Get webhook status information.
+ *
+ * @return {WebhookStatusDTO} The Webhook status information or empty body if no
+ * no settings defined.
+ */
```

Comment:
Let's add this to our docs?

Reply:
Added to the downstream PR to keep this simpler: https://github.com/metriport/metriport/pull/14/commits/e136a461c32b87d4050ae9a67af8186b38275325